rotkehlchen/tests/unit/decoders/test_makerdao_sai.py

```diff
@@ -2299,31 +2300,23 @@ def test_makerdao_sai_proxy_interaction(ethereum_transaction_decoder):
 @pytest.mark.vcr(filter_query_parameters=['apikey'])
 @pytest.mark.parametrize('ethereum_accounts', [['0xca482bCd75A6E0697aD6A1732aa187310b8372Df']])
 def test_makerdao_sai_cdp_migration(ethereum_transaction_decoder, ethereum_accounts):
+    """Data taken from
+    https://etherscan.io/tx/0x03620c6bf5edb7a7935953337ffcfac70d631cf2012d6c80d36828d636063318
+    """
+
     """Check that a Sai CDP migration is decoded properly"""
     tx_hex = deserialize_evm_tx_hash('0x03620c6bf5edb7a7935953337ffcfac70d631cf2012d6c80d36828d636063318 ')  # noqa: E501
     evmhash = deserialize_evm_tx_hash(tx_hex)
-    user_address = ethereum_accounts[0]
+    # TODO: ROTKI TEAM: The address you used here is the DSProxy address, not the user address.
```

Comment:
there may be errors in this test, but manually tracking the DSProxy is not a solution

Reply:
@LefterisJP , yes, but this is what you had already in this test case, so this is the proof that automatico recognition of DSProxy addresses is not working as you intended